### DISCUSSION TITLE
how can I concat two dataset with a sepcial ratio?

### DISCUSSION THREAD

**Discussion by crossLi** [apache/incubator-mxnet#15797] [Q&A]
I need to I concat two dataset with a sepcial ratio in some task, so how can I concat datasets and use dataloader to provide data

**Comment by vdantu** [comment]
@mxnet-label-bot add [question]
@anirudhacharya would you be able to help here?

**Comment by WenmuZhou** [comment]
Recently encountered the same problem, the data of the two data sets needs to maintain a certain proportion in each batch.

**Comment by anirudhacharya** [comment]
@crossLi the solution would be to write a custom data loader. Please check here - https://mxnet.incubator.apache.org/versions/master/tutorials/gluon/datasets.html and here - https://mxnet.incubator.apache.org/versions/master/tutorials/basic/data.html

**Comment by zachgk** [comment]
@crossLi Were you able to get the custom data loader working?

**Comment by WenmuZhou** [comment]
I have write a dataset like this and it work https://github.com/WenmuZhou/crnn.gluon/blob/master/data_loader/dataset.py#L140-L195
